src/components/DateInput/index.js

```diff
@@ -224,6 +226,7 @@ DateInput.propTypes = {
   error: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
   format: PropTypes.func,
   label: PropTypes.string,
+  locale: PropTypes.shape({}),
```

Comment:
Should be string here :)

Reply:
Locale is date-fns locale object and not currentLocale string

Reply:
Gotcha thanks, just wondering if we could use `locale.code` instead of the `currentLocale` props ?

Reply:
Yes you right useless props in this case.